lib/plugins/aws/provider/awsProvider.js

```diff
@@ -271,21 +271,26 @@ class AwsProvider {
           awsLambdaTimeout: { type: 'integer', minimum: 1, maximum: 900 },
           awsLambdaTracing: { anyOf: [{ enum: ['Active', 'PassThrough'] }, { type: 'boolean' }] },
           awsLambdaVersionning: { type: 'boolean' },
+          awsLambdaVpcConfigItems: {
+            oneOf: [
+              { type: 'string' },
+              { $ref: '#/definitions/awsCfRef' },
+              { $ref: '#/definitions/awsCfJoin' },
+              { $ref: '#/definitions/awsCfSub' },
+              { $ref: '#/definitions/awsCfImport' },
+            ],
+          },
```

Comment:
Technically it differs from `awsCfInstruction`, only by fact of not including support for `Fn::GetAtt`, it feels dubious and it doesn't feel as VPC config specific.

I think maintaince wise it'll be better to just rely on `awsCfInstruction`, also I think we can't be 100% sure, that there's no use case, where `Fn::GetAtt` can be used for VPC resolution (although I agree, it feels unlikely)

Reply:
Fair enough. I'll switch it to use the `awsCfInstruction`

Reply:
Done in a86529b